packages/contracts-bedrock/contracts/deployment/SystemDictator.sol

```diff
@@ -155,6 +155,33 @@ contract SystemDictator is OwnableUpgradeable {
         currentStep++;
     }
 
+    /**
+     * @notice Constructor required to ensure that the implementation of the SystemDictator is
+     *         initialized upon deployment.
+     */
+    constructor() {
```

Comment:
Not sure if I'm missing something, but doesn't this just make the contract unusable? Wouldn't we want to pass a `DeployConfig` into the constructor and initialize all of the values upon deployment?

Reply:
This enforces that it must be used from behind a proxy